code/messaging/src/main/java/com/adobe/marketing/mobile/aepuitemplates/uimodels/AepButton.kt

```diff
@@ -0,0 +1,33 @@
+/*
+  Copyright 2024 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+*/
+
+package com.adobe.marketing.mobile.aepuitemplates.uimodels
+
+/**
+ * Data class representing a button element in the UI.
+ *
+ * @property id The unique ID for the button.
+ * @property actionUrl The URL to be opened when the button is clicked.
+ * @property text The text to be displayed on the button, represented by an [AepText] object.
+ * @property borWidth The border width of the button.
+ * @property borColor The border color of the button represented as AepColor object.
+ * @property bgColour The background color of the button represented as AepColor object.
+ *
+ */
+data class AepButton(
+    val id: String,
+    val actionUrl: String,
+    val text: AepText,
+    val borWidth: Float? = null,
+    val borColor: AepColor? = null,
+    val bgColour: AepColor? = null,
+    val bgImg: AepImage? = null
```

Comment:
nit: unless these are being delivered as is in a network request payload can we be use the full expanded names please - `borderColor`, `backgroundColor`.